src/Interface/Modules/Render/Screenshot.cc

```diff
@@ -49,11 +49,21 @@ Screenshot::Screenshot(QOpenGLWidget *glwidget, QObject *parent)
 
 void Screenshot::takeScreenshot()
 {
-  screenshot_ = viewport_->grabFramebuffer();
+	screenshot_ = getScreenshot();
 }
 
-QImage Screenshot::getScreenshot() {
-  return viewport_->grabFramebuffer();
+QImage Screenshot::getScreenshot()
+{
+	QImage image = viewport_->grabFramebuffer();
+	for (int j = 0; j < image.height(); ++j)
+	{
+		QRgb *row = (QRgb*)image.scanLine(j);
```

Comment:
Cool fix. Let's make it more C++-like though:
```suggestion
		auto row = static_cast<QRgb*>(image.scanLine(j));
```